### ISSUE TITLE
ER: fix the name of Capitalization of one of the meetings on the events page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#6278]
### Emergent Requirement - Problem
![image](https://github.com/hackforla/website/assets/37763229/ed3534c5-c222-45ee-8984-05415033fc69)

We need to Log into VRMS and change the COMMUNITY OF PRACTICE: CoP Leads and its link https://www.hackforla.org/events/undefined) to the correct info (see resources section below)

### Issue you discovered this emergent requirement in
- #6024

### Date discovered 
2024-01-29

### Did you have to do something temporarily
- [ ] YES
- [x ] NO

### Who was involved
@ExperimentsInHonesty @JessicaLucindaCheng @roslynwythe 

### What happens if this is not addressed
It's currently listed on the Events page and then takes people to a 404 link.

### Resources
<details><summary>Content to update</summary>

#### Project Name
```
Community of Practice: CoP Leads
```
#### Project Description
```
The Leads Meeting for All the CoPs
```
#### GitHub Identifier
```
337874660
```
#### GitHub URL
```
https://github.com/hackforla/communities-of-practice
```
#### ​Slack Channel Link
```
https://hackforla.slack.com/archives/C026UMZK89X
```
#### ​Google Drive URL
```
https://drive.google.com
```
#### HFLA Website URL
```
https://www.hackforla.org/communities-of-practice
```
​
#### Partners
- none

#### Managed by Users
- none

​#### Project Status
```
Active
```
​
#### Google Drive ID
```
https://drive.google.com/drive/folders/1MfC9chrrPozQZsq02wBzXjDsQKpoPz7_
```

</details> 

<details><summary>VRMS Screenshot</summary>

![screencapture-vrms-io-projects-64f740007211620022f7757d-2024-02-11-09_54_41](https://github.com/hackforla/website/assets/37763229/c0a30ad0-452e-49dc-9abc-f6eda02c18c3)

</details> 

### Recommended Action Items
- [ ] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

### Potential solutions [draft]
I messaged VRMS team in their slack with this content on 2024-02-11
>Hi VRMS team.... I am sure there is an issue for this which 
[@Jack Haeger](https://hackforla.slack.com/team/U059005TN1L)
 can attach to this comment thread.  But I want to communicate to the whole team, that VRMS is working software.  Meaning people at Hack for LA rely on its functions.  Its no ok to roll out changes to a feature in the live environment without testing it in dev with the user.  X number of weeks ago, a change was made to the Meeting Setup/Edit page, which made some fields that we might not have data for, required (e.g., partners, Managed by Users, etc.).  This means, I cannot make updates to any projects since the change.  This is a problem that needs an urgent fix.
But the point here is not that you all put out this fire, which I do want you to do, but that you don't create new fires in the future.  If you plan to change an existing feature, that will require close coordination with the users. Right now, I can be a stand in for all users.  But things should still be coordinated with me so that the tool doesn't suddenly become unusable
tldr;
Please remove the required flag from Partners and Users in the project data.
Don't make changes to existing features without testing them with users in dev.

**Comment by ExperimentsInHonesty** [comment]
- VRMS issue https://github.com/hackforla/VRMS/issues/1577
